tests/test_port_add_remove.py

```diff
@@ -8,30 +9,50 @@
 
 """
 DELETE_CREATE_ITERATIONS defines the number of iteration of delete and create to  ports,
-we add different timeouts between delete/create to catch potential race condition that can lead to system crush.
+we add different timeouts between delete/create to catch potential race condition that can lead to system crush
+
+Add \ Remove of Buffers can be done only when the model is dynamic.
 """
 DELETE_CREATE_ITERATIONS = 10
 
+@pytest.yield_fixture
+def dynamic_buffer(dvs):
+    buffer_model.enable_dynamic_buffer(dvs.get_config_db(), dvs.runcmd)
+    yield
+    buffer_model.disable_dynamic_buffer(dvs.get_config_db(), dvs.runcmd)
+
+@pytest.mark.usefixtures('dvs_port_manager')
+@pytest.mark.usefixtures("dynamic_buffer")    
 class TestPortAddRemove(object):
 
     def test_remove_add_remove_port_with_buffer_cfg(self, dvs, testlog):
         config_db = dvs.get_config_db()
         asic_db = dvs.get_asic_db()
+        state_db = dvs.get_state_db()
+        app_db = dvs.get_app_db()
+
+        # set mmu size
+        fvs = {"mmu_size": "12766208"}
+        state_db.create_entry("BUFFER_MAX_PARAM_TABLE", "global", fvs)
+        bufferMaxParameter = state_db.wait_for_entry("BUFFER_MAX_PARAM_TABLE", PORT_A)
```

Comment:
We should not assume the test `test_remove_add_remove_port_with_buffer_cfg` will alway run at first.
So let's move L35~36 to a commonplace, something like `setup_db`, and remove L37 which is unnecessary.
Otherwise, it will fail if we run other test cases without this one.